.github/workflows/unit_test.yml

```diff
@@ -25,17 +25,6 @@ jobs:
           name: test-report-unit
           path: /Users/runner/Library/Developer/Xcode/DerivedData/*/Logs/Test/*.xcresult
           
-      - name: unit-test
-        run:
-          xcodebuild test -workspace CardScan.xcworkspace -scheme CardScanTests -destination 'platform=iOS Simulator,name=iPhone 11'
-
-      - name: upload-artifacts-unit
-        uses: actions/upload-artifact@v2
-        if: failure()
-        with:
-          name: test-report-unit
-          path: /Users/runner/Library/Developer/Xcode/DerivedData/*/Logs/Test/*.xcresult
-          
```

Comment:
this will prevent unit tests from running. is this what we want?

Reply:
All the unit tests (including the new payload test) are in `CardScanExampleTests`. Its okay to remove the `CardScanTests` scheme!

Reply:
The new unit test among the older tests
<img width="1239" alt="Screen Shot 2021-05-05 at 3 50 02 PM" src="https://user-images.githubusercontent.com/42391849/117219223-b4defe00-adb9-11eb-9c58-9c05da81e9f2.png">